src/search-bar/search.css

```diff
@@ -0,0 +1,64 @@
+
+@import 
+url('https://fonts.googleapis.com/css2?family=Rubik:ital,wght@0,300..900;1,300..900&display=swap');
+
+.logo{
```

Comment:
not mandatory but using a very general class name makes the code harder to read.